src/components/task/create-new/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import Button from "components/common/button";
+import TextArea from "components/common/text-area";
+import Image from "components/common/image";
+import { addTaskToTodo } from "store/actions/todo";
+import { generateTaskObject, sanitizer } from "utils/helper";
+import { ERROR_MESSAGE_TASK_TITLE } from "utils/const/errorMessages";
```

Comment:
use proper file naming convention.

```suggestion
import { ERROR_MESSAGE_TASK_TITLE } from "utils/const/error-messages";
```